.github/workflows/main.yaml

```diff
@@ -1,7 +1,7 @@
 name: Native Build & Test
 
 env:
-  cacheId: "6" # increment to expire the cache
```

Comment:
Is this cacheid still needed? GitHub now allow admins to manage cache and invalidate them via UI

Reply:
Probably not then. I'll remove it via a different PR.